doc/source/mast_data_products/hap-parameters.rst

```diff
@@ -410,7 +446,7 @@ generate_source_catalogs (*primarily in align_utils.py*)
 """"""""""""""""""""""""""""""""""""""""""""""""""""""""
```

Comment:
Add a note so the user is reminded...The generate_source_catalogs section is used for alignment purposes only. This has nothing to do with the output HAP catalog products.

Reply:
I've added them as a note now. I agree that it's not the ideal place, but I couldn't find a better place. With a "note", it seems to flow better now.